src/core/swa-cli-persistence-plugin/impl/machine-identifier.ts

```diff
@@ -22,10 +22,10 @@ export async function getMachineId(): Promise<string> {
  * Get the mac address of the machine and hash it.
  * @returns {Promise<string>} A 32-byte hash of the mac address.
  */
```

Comment:
32 bytes.. is the jsdoc correct ?Shall we update it ?

Reply:
Wassim mentioned he'll be updating the comments